src/jit/compiler.h

```diff
@@ -4266,8 +4266,24 @@ class Compiler
     // the call may modify the heap (we assume arbitrary memory side effects if so).
     bool fgValueNumberHelperCall(GenTreeCall* helpCall);
 
-    // Requires "helpFunc" to be pure.  Returns the corresponding VNFunc.
-    VNFunc fgValueNumberHelperMethVNFunc(CorInfoHelpFunc helpFunc);
+    // Requires that "helpFunc" is one of the pure Jit Helper methods.
+    // Returns the corresponding VNFunc to use for value numbering
+    VNFunc fgValueNumberJitHelperMethodVNFunc(CorInfoHelpFunc helpFunc);
```

Comment:
Consistency with existing code can be a good thing but maybe it's time to stop using `fg` for everything. All these new functions should have a `vn` prefix.

Reply:
Yeah, I know,  I can do that in an upcoming changeset

Reply:
OK I will change the names of these methods to start with vnAddExceptionSet